cmd/dockervolumed/managedplugin/README.md

```diff
@@ -76,51 +76,77 @@ Debian 9.x (stable):
 ```apt-get install -y open-iscsi multipath-tools xfsprogs
 modprobe xfs
 sed -i"" -e "\$axfs" /etc/modules
-docker plugin install --disable --grant-all-permissions --alias nimble store/nimblestorage/nimble:2.5.1
+docker plugin install --disable --grant-all-permissions --alias nimble store/hpestorage/nimble:3.0.0
 docker plugin set nimble PROVIDER_IP=192.168.171.74 PROVIDER_USERNAME=admin PROVIDER_PASSWORD=admin iscsiadm.source=/usr/bin/iscsiadm glibc_libs.source=/lib/x86_64-linux-gnu
 docker plugin enable nimble
 systemctl daemon-reload
 systemctl restart open-iscsi multipath-tools
 ```
 
+**NOTE:** To use the plugin on Fibre Channel environment use `PROTOCOL=FC` environment variable.
+
 ### Making Changes
 
 The `docker plugin set` command can only be used on the plugin if it is disabled. To disable the plugin, use the `docker plugin disable` command. For example:
 
-```$ docker plugin disable nimble
+```markdown
+
+$docker plugin disable nimble
 ```
 
+#### Settable Parameters
+
+List of parameters which are supportable to be settable by the plugin
```

Comment:
nit: "supportable to be settable" doesn't read right.

Reply:
will change to supported to be settable